early-attack-detector-py/src/agent.py

```diff
@@ -275,12 +298,13 @@ def detect_malicious_contract(
             function_signatures = get_function_signatures(w3, opcodes)
             logger.info(f"{created_contract_address}: score={model_score}")
 
-            if model_score is None or model_score < MODEL_INFO_THRESHOLD:
+            if (model_score is None or model_score < MODEL_INFO_THRESHOLD) and not from_eoa_in_tp_list:
```

Comment:
Perhaps could be refactored to
```suggestion
            if not from_eoa_in_tp_list:
                if model_score is None or model_score < MODEL_INFO_THRESHOLD:
                    if ENV == 'dev':
                        logger.info(f"Score is less than threshold: {model_score} < {MODEL_INFO_THRESHOLD}. Not creating alert.")
                    return []
                # If we are not in beta, we create alerts if the score is above the threshold
                # or if `from_` is in the True Positive list
                if model_score < MODEL_THRESHOLD and not BETA:
                    if ENV == 'dev':
                        logger.info(f"Score is less than threshold: {model_score} < {MODEL_THRESHOLD} and we are not in beta. Not checking for labels.")
                    return []
```